mlrun/common/constants.py

```diff
@@ -14,6 +14,7 @@
 #
 IMAGE_NAME_ENRICH_REGISTRY_PREFIX = "."  # prefix for image name to enrich with registry
 MLRUN_CREATED_LABEL = "mlrun-created"
+MLRUN_FUNCTIONS_LABEL = "mlrun-functions"
```

Comment:
1. This is used for an annotation, not a label - so `MLRUN_FUNCTIONS_ANNOTATION`
2. In nuclio we usually annotate internal stuff with a `nuclio.io/` prefix. In mlrun we don't really do it but we do label internal stuff with `mlrun/` prefix so I think we should go with that in the annotation as well - `mlrun/function-names`